README.md

```diff
@@ -27,8 +27,8 @@ Tools built on DynamoRIO include:
   [drcpusim](http://dynamorio.org/docs/page_drcpusim.html)
 - The "strace for Windows" tool [drstrace](http://drmemory.org/strace_for_windows.html)
 - The code coverage tool [drcov](http://dynamorio.org/docs/page_drcov.html)
-- The library tracing tool [drltrace](http://drmemory.org/docs/page_drltrace.html)
```

Comment:
Ah I see. Would it make sense to set the link to https://github.com/DynamoRIO/drmemory/tree/master/drltrace until the drmemory.org link works?